maskit/utils.py

```diff
@@ -6,6 +6,10 @@
 def check_params(train_params):
     assert train_params["dataset"] in ["simple", "iris", "mnist", "circles"]
     assert isinstance(train_params["optimizer"], ExtendedOptimizers)
+    if "interpret" not in train_params:
+        train_params["interpret"] = tuple(
+            range(2 ** len(train_params.get("wires_to_measure", (0,))))
```

Comment:
No idea whether you agree, but I got confused mentally parsing all those ``)``s... :P
```suggestion
            range(2 ** len(train_params.get("wires_to_measure", [0])))
```